ui/src/app/shared/model/about/feature-info.model.ts

```diff
@@ -52,12 +57,12 @@ export class FeatureInfo implements Serializable<FeatureInfo> {
           return this.skipperEnabled;
         }
         default: {
-          console.error(`Unsupported feature parameter '${feature}'.`);
+          // console.log.error(`Unsupported feature parameter '${feature}'.`);
```

Comment:
@oodamien How do we want to handle this use-case, of logging from within domain classes? Not super-common, but model/domain classes may contain logic that may require logging.

In Java we have Slf4j, log4j etc. that can be used statically, not as a service. Feels odd to inject a service here. Was there an issue that you commented this out?

Also saw this open Angular issue ... https://github.com/angular/angular/issues/5458

Reply:
@oodamien Please see this commented out log statement